net.sf.VICE.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">net.sf.VICE.desktop</id>
+  <name>VICE</name>
+  <summary>Versatile Commodore Emulator</summary>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
If you are the author of this file, we prefer `CC0-1.0`.

Reply:
Done.